VideoAPI/VideoApi.IntegrationTests/Database/Queries/GetMessagesForConferenceQueryTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Linq;
+using Faker;
+using FluentAssertions;
+using NUnit.Framework;
+using Testing.Common.Helper.Builders.Domain;
+using VideoApi.DAL;
+using VideoApi.DAL.Exceptions;
+using VideoApi.DAL.Queries;
+using VideoApi.Domain.Enums;
+using Task = VideoApi.Domain.Task;
```

Comment:
why are you using a task? this shouldn't be neeed for this test

Reply:
Removed